### PR TITLE
Hotfix/allow starRates lower 0

### PR DESCRIPTION
**Pull request by gorgos**
High-prio

@gustavoguimaraes Please review asap.

### CODE REVIEW

File: contracts/TokenSale.sol [hunk 1]
```diff
@@ -197,13 +197,13 @@ contract TokenSale is FinalizableCrowdsale, Pausable {
         uint256 starAllocationToTokenSale = starToken.allowance(beneficiary, this);
         if (starAllocationToTokenSale > 0) {
             // calculate token amount to be created
-            uint256 tokens = starAllocationToTokenSale.mul(starRate);
+            uint256 tokens = starAllocationToTokenSale.mul(starRate).div(1000);
```
Comment (gustavoguimaraes) [inline contracts/TokenSale.sol]: Why 1000?

Comment (gustavoguimaraes) [reply contracts/TokenSale.sol]: Should be 100

Comment (gorgos) [reply contracts/TokenSale.sol]: Doesn't matter what we insert here as long as the passed `starRate` is accordingly. Maybe I should add a comment to the init function@_starRate param.

Comment (gorgos) [reply contracts/TokenSale.sol]: 1000 allows us to accept a bit lower `starRates` than just 100.

**Pull request merged**